lte/gateway/c/session_manager/test/test_set_session_manager_handler.cpp

```diff
@@ -132,40 +127,45 @@ class SessionManagerHandlerTest : public ::testing::Test {
         magma::NotifyUeEvents::PDU_SESSION_INACTIVE_NOTIFY);
 
     reqcmn->mutable_sid()->set_id("IMSI000000000000001");
-    reqcmn->set_apn("BLR");
+    reqcmn->set_apn("example.com");
     reqcmn->set_ue_ipv4("192.168.128.11");
     reqcmn->set_rat_type(magma::RATType::TGPP_NR);
     reqcmn->set_sm_session_state(magma::SMSessionFSMState::CREATING_0);
+  }
 
-    auto imsi   = reqcmn->sid().id();
-    auto apn    = reqcmn->apn();
-    auto pdu_id = req->pdu_session_id();
-    auto cfg = set_session_manager->m5g_build_session_config(*session_ctx_req);
-    auto& noti    = *req;
-    auto ue_event = noti.notify_ue_event();
-    std::function<void(Status, SmContextVoid)> response_callback;
-
-    EXPECT_EQ(pdu_id, 0x5);
-    EXPECT_EQ(imsi, "IMSI000000000000001");
-    EXPECT_EQ(apn, "BLR");
+  void assert_sm_session_context(magma::SetSMSessionContext* request) {
```

Comment:
I am still confused with what are we really asserting here. Why do we need a test to assert an object we created on a test isself? 

In my opinion this test case its only testing Googles GRPC code, but not magma